site/docs/releases.md

```diff
@@ -68,6 +68,8 @@ High-level features:
   * Added predicate pushdown for partitions and files metadata tables [[\#2358](https://github.com/apache/iceberg/pull/2358), [\#2926](https://github.com/apache/iceberg/pull/2926)].
   * Added a new, more flexible compaction action for Spark that can support different strategies such as bin packing and sorting. [[\#2501](https://github.com/apache/iceberg/pull/2501), [\#2609](https://github.com/apache/iceberg/pull/2609)].
   * Added the ability to upgrade to v2 or create a v2 table using the table property format-version=2  [[\#2887](https://github.com/apache/iceberg/pull/2887)].
+* **Hive**
+  * Added the ability to set the catalog at the table level in the Hive Metastore. This makes it possible to write queries that reference tables from multiple catalogs [[\#2129](https://github.com/apache/iceberg/pull/2129)].
```

Comment:
Configuring catalog using `iceberg.mr.catalog` is now deprecated. (https://github.com/apache/iceberg/pull/2565)

Multi table insert support is added (no multi-table ACID transaction guarantee) (https://github.com/apache/iceberg/pull/2228)

Reply:
Hive: Vectorized ORC reads for Hive (#2613)

Reply:
@danielcweeks Hive vectorized reads were already mentioned in the "Other notable changes" section. I moved them to the high-level changes section.